packages/ra-tree-ui-materialui/src/constants.js

```diff
@@ -0,0 +1 @@
+export const DROP_TARGET_TYPE = 'RA/TREE/NODE';
```

Comment:
this should be in `ra-tree-core`, no?

Reply:
I think not, not all UI implementations might allow drag & drop